libs/libarchfpga/src/read_xml_arch_file.cpp

```diff
@@ -3639,6 +3639,12 @@ static void ProcessSegments(pugi::xml_node Parent,
 
     /* Load the segments. */
     Node = get_first_child(Parent, "segment", loc_data);
+
+    /*Keep track of the type of axis of the segments loaded*/
```

Comment:
Can improve this comment.  Flags to see if we have *any* x- and y-directed segment types specified, respectively.